gromacs/tools.py

```diff
@@ -76,14 +76,53 @@ def _generate_sphinx_class_string(clsname):
 # class g_dist(GromacsCommand):
 #     command_name = 'g_dist'
 
+aliases5to4 = {
+    'grompp': 'grompp',
+    'eneconv': 'eneconv',
+    'sasa': 'g_sas',
+    'convert_tpr': 'tpbconv',
+    'editconf': 'editconf',
+    'pdb2gmx': 'pdb2gmx',
+    'trjcat': 'trjcat',
+    'trjconv': 'trjconv',
+    'trjorder': 'trjorder',
+    'trjorder': 'trjorder',
+    'xpm2ps': 'xpm2ps',
+    'mdrun': 'mdrun',
+    'make_ndx': 'make_ndx',
+    'make_edi': 'make_edi',
+    'gmxdump': 'gmxdump',
+    'gmxcheck': 'gmxcheck',
+    'genrestr': 'genrestr',
+    'genion': 'genion',
+    'genconf': 'genconf',
+    'do_dssp': 'do_dssp'    
+}
+
 for name in sorted(config.load_tools):
     # hack for 5.x 'gmx toolname': add as gmx:toolname
     if name.startswith('gmx:'):
         name = name[4:]
+        #make alias for backwards compatability
+        #checka agianst uncommon name changes
+        #have to check each one, since it's possible there are suffixes like for double precision
+        flag = False
```

Comment:
This is a bit ugly, can't you split off suffixes, test the stem with a dict lookup and then attach the suffix again?

And the current loop could be written
```python
# flag = False # not needed
for c5, c4 in aliases5to4.items():
    if name.startswith(c5):
        old_name = c4 + name.split(c5)[1]
        break
else:
   old_name = 'g_' + name
```